google/cloud/firestore_v1/transaction.py

```diff
@@ -136,7 +140,7 @@ def _commit(self) -> list:
         self._clean_up()
         return list(commit_response.write_results)
 
-    def get_all(self, references) -> Any:
+    def get_all(self, references) -> Coroutine:
```

Comment:
Should be a generator:
```suggestion
    def get_all(self, references) -> Generator[DocumentSnapshot, Any, Any]:  # send type None?
```